.github/workflows/post-apiview.yml

```diff
@@ -0,0 +1,27 @@
+name: After APIView
+
+on:
+  check_suite:
+    types: [completed]
+    
+permissions:
+  pull-requests: write
+  contents: read
+
+jobs:
+  event-handler:
+    name: After APIView
+    runs-on: ubuntu-24.04
+    if: github.event.check_suite.app.name == 'Azure Pipelines'
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          sparse-checkout: 'eng/common'
+          fetch-depth: 0
```

Comment:
```suggestion
```

fetch-depth:0 actually means "all history".  You probably want fetch-depth:1, which is also the default, so you can omit it.